packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -68,13 +68,25 @@ const MembersView = () => {
   const router = useRouter();
   const session = useSession();
   const utils = trpc.useContext();
+
   const teamId = Number(router.query.id);
 
   const showDialog = router.query.inviteModal === "true";
   const [showMemberInvitationModal, setShowMemberInvitationModal] = useState(showDialog);
   const [showInviteLinkSettingsModal, setInviteLinkSettingsModal] = useState(false);
 
-  const { data: team, isLoading } = trpc.viewer.teams.get.useQuery(
+  const { data: orgMembersNotInThisTeam, isLoading: isOrgListLoading } =
+    trpc.viewer.organizations.getMembers.useQuery(
+      {
+        teamIdToExclude: teamId,
+        distinctUser: true,
```

Comment:
We pass in distinct here as we dont want multiple of the same user ID to be pulled back

Reply:
<img width="679" alt="Screenshot 2023-06-20 at 5 18 58 PM" src="https://github.com/calcom/cal.com/assets/53316345/20d69918-d4f3-4790-a310-048357bd2f22">

i keep getting too many 404 in this query